redi-connect-front/src/pages/app/me/Me.tsx

```diff
@@ -188,17 +174,19 @@ const validationSchema = Yup.object({
   })
 })
 
-const mapState = (state: RootState) => ({
-  saveResult: state.user.saveResult
-})
+const Me = ({ loading, profile, saveResult, profileFetchStart, profileSaveStart }: any) => {
+  // not sure if this is really neede since the profile is loaded when the user is logged in
+  useEffect(() => {
```

Comment:
Let's keep it in here for now. Better be explicit about loading the profile here. It's rather ther "load profile on every page change" mechanism that should be changed.

Once we get around to updating the data/store layer I 'll also want to at least use the new hooks that come with redux to get the profile object. There's very likely even better ways to do it.

Reply:
sure, I foucus on simplifying the overall structure, after that we can talk details.